src/test/rgw/CMakeLists.txt

```diff
@@ -63,3 +63,29 @@ target_link_libraries(ceph_test_rgw_obj
 set_target_properties(ceph_test_rgw_obj PROPERTIES COMPILE_FLAGS
   ${UNITTEST_CXX_FLAGS})
 
+# ceph_test_rgw_bencode
+set(test_rgw_bencode_srcs test_rgw_bencode.cc)
+add_executable(ceph_test_rgw_bencode
+  ${test_rgw_bencode_srcs}
+  )
+target_link_libraries(ceph_test_rgw_bencode
+  rgw_a
+  cls_rgw_client
+  cls_lock_client
+  cls_refcount_client
+  cls_log_client
+  cls_statelog_client
+  cls_version_client
+  cls_replica_log_client
+  cls_user_client
+  librados
+  global
+  curl
+  uuid
+  expat
+  ${CMAKE_DL_LIBS}
+  ${UNITTEST_LIBS}
+  ${CRYPTO_LIBS}
+  )
+set_target_properties(ceph_test_rgw_bencode PROPERTIES COMPILE_FLAGS
+  ${UNITTEST_CXX_FLAGS})
```

Comment:
I have updated my code. but the unittest_rgw_bencode can not generate when I run make check.
I modify add  "#define ENABLE_CLIENT 1" in src/acconfig.h and run make check again. the unittest_rgw_bencode can not generate. I think there are something configrations that I have ignored. I want to know why unittest_rgw_bencode can not be produced? Is it wrong with my code? thanks!